invenio_notify/records/models.py

```diff
@@ -0,0 +1,51 @@
+from invenio_db import db
+from sqlalchemy import or_
+from sqlalchemy_utils.models import Timestamp
+
+from invenio_notify.errors import NotExistsError
+
+
+class NotifyInboxModel(db.Model, Timestamp):
```

Comment:
Of course we need created. But I'm not sure we'll need updated.

Reply:
Timestamp mixin introduces created and updated. Do we need updated?

Reply:
yes

Reply:
why?

Reply:
you will know that in the future.
even you don't trust me or no update feature for the model now, 
applying Timestamp quite standard in system.